src/persist-client/src/impl/machine.rs

```diff
@@ -1078,7 +1096,7 @@ mod tests {
                             let writer_id = write.writer_id.clone();
                             let (_, _) = write
                                 .machine
-                                .compare_and_append(batch, &writer_id)
+                                .compare_and_append(batch, &writer_id, now())
```

Comment:
looks like this one can also be `(client.cfg.now)()`

Reply:
👍

tagging https://github.com/MaterializeInc/materialize/pull/13847 -- there's at least a bundle of future improvements to leases I'll group this in wtih: making sure the clocks are all right in tests, rewriting reader leases to look more like writer leases (but duration in state), and the reader regression test